unit_tests/src/test_re_serialization.cpp

```diff
@@ -308,46 +308,42 @@ TEST_CASE("serialize_dataObjInfo_ptr", "[pointer][serialization]")
 
     SECTION("empty and basic_struct")
     {
-        std::map<std::string_view, DataObjInfo> test_data{
-            {"empty", {}},
-            {"non_empty", {
-                .objPath            = "objPath",
-                .rescName           = "rescName",
-                .rescHier           = "rescHier",
-                .dataType           = "dataType",
-                .dataSize           = 100,
-                .chksum             = "chksum",
-                .version            = "version",
-                .filePath           = "filePath",
-                .dataOwnerName      = "dataOwnerName",
-                .dataOwnerZone      = "dataOwnerZone",
-                .replNum            = 200,
-                .replStatus         = 300,
-                .statusString       = "statusString",
-                .dataId             = 400,
-                .collId             = 500,
-                .dataMapId          = 600,
-                .flags              = 700,
-                .dataComments       = "dataComments",
-                .dataMode           = "dataMode",
-                .dataExpiry         = "dataExpiry",
-                .dataCreate         = "dataCreate",
-                .dataModify         = "dataModify",
-                .dataAccess         = "dataAccess",
-                .dataAccessInx      = 800,
-                .writeFlag          = 900,
-                .destRescName       = "destRescName",
-                .backupRescName     = "backupRescName",
-                .subPath            = "subPath",
-                .specColl           = nullptr,
-                .regUid             = 1000,
-                .otherFlags         = 1100,
-                .condInput          = {},
-                .in_pdmo            = "in_pdmo",
-                .next               = nullptr,
-                .rescId             = 1200
-            }}
-        };
+        std::map<std::string_view, DataObjInfo> test_data{{"empty", {}},
+                                                          {"non_empty",
+                                                           {.objPath = "objPath",
+                                                            .rescName = "rescName",
+                                                            .rescHier = "rescHier",
+                                                            .dataType = "dataType",
+                                                            .dataSize = 100,
+                                                            .chksum = "chksum",
```

Comment:
Let's keep the original formatting for this and wrap it in `clang-format off/on`.

Reply:
Done.